docs/source/kalman_objective.md

```diff
@@ -0,0 +1,481 @@
+
+
+## Objectives for Kalman-type ensembles
+
+When constraining $Q$ to only encode information on the two first moments, that are learned from data, the KLD
+approach yields a Gaussian distribution $Q$ even when $P$ is non-Gaussian.
+We call methods using the Kalman-gain in Gaussian transport from prior to posterior as "Kalman-type" ensemble based data assimilation methods.
+They implicitly seek to minimize information loss in this transport.
+Different Kalman-type modelling approaches essentially yields different estimates of the Kalman-gain, $K$.
+The following discuss how to evaluate (on some objective, using unseen test-data)
+which Kalman-gains are superior to others.
+This turns out to be non-trivial for several common applications like the EnKF/ES due to implicit singular covariance estimates.
+
+
+### Joint-Gaussian
+The KLD approach suggests $-E_P[\log(Q)]$ as the objective of fitting $Q$.
+Taking $Q$ to be multivariate Gaussian, we arrive at the negative log-likelihood for a sample
+
+$$
+-\log p(x, y) = \frac{1}{2} \log \left| \begin{bmatrix}
+\Sigma_{x} & \Sigma_{xy} \\
+\Sigma_{yx} & \Sigma_{y}
+\end{bmatrix} \right| + \frac{k}{2} \log(2\pi) + \frac{1}{2} \begin{bmatrix}
+x - \mu_x \\
+y - \mu_y
+\end{bmatrix}^T \begin{bmatrix}
+\Sigma_{x} & \Sigma_{xy} \\
+\Sigma_{yx} & \Sigma_{y}
+\end{bmatrix}^{-1} \begin{bmatrix}
+x - \mu_x \\
+y - \mu_y
+\end{bmatrix}
+$$
+
+and the negative log-likelihood 
+
+$$
+-\frac{1}{n}\sum_i \log(p(x_i,y_i))
+$$
+
+as an estimator of $-E_P[\log(Q)]$, suitable for fitting procedures.
+It may also be used for evaluation of different models, as long as the sample $(x_i,y_i)$ is not used to optimize $\theta$ parametrizing $Q$.
+This means we may use the same formula, but with a different test-set of samples.
+This is a natural objective if explicit estimates of $\Sigma_{x}$, $\Sigma_{xy}$, and $\Sigma_{y}$ are computed in creating
+an estimate $K=\Sigma_{xy}\Sigma_{y}^{-1}$.
+Importantly, it allows encoding information or structure of how $\Sigma_{x}$, $\Sigma_{xy}$, and $\Sigma_{y}$ should look like,
+for instance  $\Sigma_y=H\Sigma_xH^T+\Sigma_{\epsilon}$. Then only $\Sigma_x$ and $H$ needs estimation if they should be unknown.
+It could also be that $\Sigma_x$ is diagonal, or is a function of some $\theta$ in a smaller dimensional space.
+We would then require fewer degrees of freedom to fit $\Sigma_x(\theta)$.
+
+Note that if covariance estimates are singular (i.e. EnKF when $p>n$) then this objective cannot be used due to the log-determinant and covariance inverse.
+Furthermore, we often only have access to the final estimate of the Kalman-gain, not the full covariance structure.
+
+### Gaussian conditional marginal
+
+The joint KLD is a natural objective to evaluate model performance on.
+However, multiple (Kalman) ensemble methods only produce an estiamte $K$ either explicit or explicit,
```

Comment:
```suggestion
However, multiple (Kalman) ensemble methods only produce an estimate $K$ either explicit or implicit,
```

Not sure I understand this sentence.
Do you mean that several different ensemble methods only produce an estimate of $K$ without specifying covariance structure?
What do you mean by explicit / implicit?